prime-router/src/main/kotlin/history/azure/DatabaseDeliveryAccess.kt

```diff
@@ -141,4 +144,47 @@ class DatabaseDeliveryAccess(
             query.fetchInto(DeliveryFacility::class.java)
         }
     }
+
+    /**
+     * Get a list of matching delivery facilities for a set of [reportIds]
+     * @link https://github.com/CDCgov/prime-reportstream/issues/9272
+     */
+    fun fetchBulkDeliveryFacilities(
+        reportIds: List<ReportId>
+    ): List<DeliveryFacility> {
+        val positive: Field<String> = DSL.field(
+            "sum(CASE WHEN test_result = 'DETECTED' THEN 1 ELSE 0 END)",
+            String::class.java
+        ).`as`("positive")
+        val total: Field<String> = DSL.field(
+            "count(covid_results_metadata_id)",
+            String::class.java
+        ).`as`("count_records")
+
+        return db.transactReturning { txn ->
+            val query = DSL.using(txn)
+                .select(
+                    COVID_RESULT_METADATA.TESTING_LAB_NAME,
```

Comment:
@penny-lischer can you confirm these 4 items are all you need for the facilities endpoint?

Reply:
@arnejduranovic I also need
created_at
sender_id

Reply:
Updated to include all the data needed for the design

Reply:
@TurkeyFried please update swagger to reflect changes.

Reply:
Swagger should be good now